run-tests.php

```diff
@@ -3672,20 +3666,6 @@ public function __construct(array $environment, string $tool = 'memcheck')
         $this->header = sprintf("%s (%s)", trim($header), $this->tool);
         $this->version_3_8_0 = version_compare($version, '3.8.0', '>=');
     }
-
-    public function wrapCommand(string $cmd, string $memcheck_filename, bool $check_all): string
```

Comment:
never invoked method.

in the past this was used in https://github.com/php/php-src/blob/196f8fdfc3e792d316311b55e50a4ae4c97c5b71/run-tests.php#L2432-L2437

Reply:
Why do you think this `if` code can no longer be executed? Or _how_ did you conclude this is never invoked?

Reply:
good catch!

phpstorm indicated the unused code. it seems it got tricked by the untyped `global` variables.

would you accept a new PR which gets rid of global variables in run-tests where possible?

Reply:
Maybe that's not worth it, it will make merging fixes in the test runner from lower to higher branches a bit harder, for no obvious benefit imho.

Reply:
I see. proposed something in https://github.com/php/php-src/pull/16678 - feel free to close it if you don't think its valuable